firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -119,12 +134,9 @@ uint8_t TCMD_execute_parsed_telecommand_now(const uint16_t tcmd_idx, const char
     }
     TCMD_TelecommandDefinition_t tcmd_def = TCMD_telecommand_definitions[tcmd_idx];
 
-    DEBUG_uart_print_str("=========================");
```

Comment:
Please revert these changes in this PR. They're unnecessary (and maybe a regression).

Reply:
Sure thing. I just copied what is originally in main into this